capstone/src/main/webapp/authenticate.js

```diff
@@ -12,23 +13,14 @@ window.addEventListener('load', handleClientLoad);
 */
 function createDynamicSigninButton() {
     var dynamicButton = document.createElement("div");
-    var loginButton = dynamicButton.appendChild(createLoginButton());
     var logoutButton = dynamicButton.appendChild(createLogoutButton());
-    return [dynamicButton, loginButton, logoutButton];
```

Comment:
At this point dynamic button isnt really dynamic anymore since it is always a logout button right? Because of that, the variable name should be updated, but actually since all it does is append a single child logout button, the places that use it should really just use the logout button and remove the variable altogether